src/MenuItem.cpp

```
@@ -0,0 +1,7 @@
+#include "MenuItem.h"
+
+bool MenuItem::_isEditing = false;
+
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)